pkg/controller/service/service_controller.go

```diff
@@ -60,13 +60,19 @@ const (
 	minRetryDelay = 5 * time.Second
 	maxRetryDelay = 300 * time.Second
 
-	// LabelNodeRoleMaster specifies that a node is a master
-	// It's copied over to kubeadm until it's merged in core: https://github.com/kubernetes/kubernetes/pull/39112
-	LabelNodeRoleMaster = "node-role.kubernetes.io/master"
-
-	// LabelNodeRoleExcludeBalancer specifies that the node should be
-	// exclude from load balancers created by a cloud provider.
-	LabelNodeRoleExcludeBalancer = "alpha.service-controller.kubernetes.io/exclude-balancer"
+	// labelNodeRoleMaster specifies that a node is a master. This label is deprecated and
```

Comment:
I don't think "deprecated" captures what you wrote in the KEP; propose "the use of this label to determine whether a node is attached to a loadbalancer is deprecated"

Reply:
Agreed, clarified.